tests/amqpvalue_ut/amqpvalue_ut.c

```diff
@@ -124,7 +124,7 @@ static void* test_context = (void*)0x4243;
 
 static TEST_MUTEX_HANDLE g_testByTest;
 
-MU_DEFINE_ENUM_STRINGS(UMOCK_C_ERROR_CODE, UMOCK_C_ERROR_CODE_VALUES)
+MU_DEFINE_ENUM_STRINGS_WITHOUT_INVALID(UMOCK_C_ERROR_CODE, UMOCK_C_ERROR_CODE_VALUES)
```

Comment:
>MU_DEFINE_ENUM_STRINGS_WITHOUT_INVALID [](start = 0, length = 38)

Sorry but I do not see the reason to switch back. Using the invalid value actually caught unintended comparisons. There is no real overhead in using it.